### ISSUE TITLE
Questions about video training

### ISSUE THREAD

**Issue by ander008** [m87-labs/moondream#144]
Will future versions of moondream support video training?

**Comment by vikhyat** [comment]
Yes, we still have work to do on image understanding but video is also planned for the future. What sorts of use-cases are you interested in?

**Comment by Potatooff** [comment]
security camera can be a good one!

**Comment by torayeff** [comment]
Any update on video understanding?
